ci/common/prep-init-env.sh

```diff
@@ -57,5 +57,6 @@ fi
 
 # Cleanup local git
 cd $1
-git clean -xdf
+# Exclude folder for running local CI tests
+git clean -xdf -e ci_binded/
```

Comment:
It looks like this folder is generated by the local CI run within the repo, but it's empty (ls ci_binded/build shows nothing). Does it get cleaned after the test? Can it be removed?

Reply:
@tijyojwad, <repp>/ci_binded is mounted to <repo>/<docker_image_tag_name>/cpp. Once  <repo>/<docker_image>/cpp content is deleted by the git clean command, the folder remains empty. This mounted directory allows using the same BUILD_SCRIPT content for different images. We do not really need this, because we anyway clean the environment before, Or with could use a symbolic link.

Reply:
@ohadmo I see, okay not a big deal if it stays around since it doesn't show up on `git status` and so won't get checked in by mistake.

Reply:
@tijyojwad I removed the mounted directory creation, we do not need it.

Reply:
Sweet!